requirements.txt

```diff
@@ -4,5 +4,6 @@ numpy
 phasespace
 progress
 pyyaml
+sympy
```

Comment:
Should also be added to `setup.py` (which is used by PyPI)

Reply:
1bf2ee2ba9cb0527d925a6388fa2b82a835e51e2